src/Http/ApiProblem/ApiProblem.php

```diff
@@ -303,6 +304,15 @@ public static function bodyMissing(): self
         );
     }
 
+    public static function unsupportedMediaType(): self
+    {
+        return self::create(
+            'https://docs.publiq.be/docs/errors/baae4bf3cece5-header#unsupported-media-type',
```

Comment:
You should be able to use `https://api.publiq.be/probs/header/unsupported-media-type` here